File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--add-chart-controls-to-worksheets_1.cs
            Microsoft.Office.Tools.Excel.Chart employeeData;
            employeeData = this.Controls.AddChart(25, 110, 200, 150, "employees");
            employeeData.ChartType = Excel.XlChartType.xl3DPie;

            // Gets the cells that define the data to be charted.
            Excel.Range chartRange = this.get_Range("A5", "D8");
            employeeData.SetSourceData(chartRange, missing);

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--add-chart-controls-to-worksheets_1.vb
        Dim employeeData As Microsoft.Office.Tools.Excel.Chart
        employeeData = Me.Controls.AddChart(25, 110, 200, 150, "employees")
        employeeData.ChartType = Excel.XlChartType.xl3DPie

        ' Gets the cells that define the data to be charted.
        Dim chartRange As Excel.Range = Me.Range("A5", "D8")
        employeeData.SetSourceData(chartRange)